versioned_docs/version-1.2.0/release-notes/release-notes-for-kubeslice-oss-1.2.0.mdx

```diff
@@ -0,0 +1,24 @@
+# Release Notes for KubeSlice OSS 1.2.0
+
+*Release date: 9th Jan 2024*
```

Comment:
This date needs to be changed to the actual release date.

Reply:
done